clang/test/Analysis/Checkers/WebKit/ref-cntbl-crtp-base-no-virtual-dtor.cpp

```diff
@@ -119,6 +119,11 @@ template<class T, DestructionThread destructionThread = DestructionThread::Any>
             ensureOnMainThread([this] {
                 delete static_cast<const T*>(this);
             });
+        } else if constexpr (destructionThread == DestructionThread::MainRunLoop) {
+            auto deleteThis = [this] {
```

Comment:
Can this variable be declared as `Function<>` instead? (It's probably unlikely. But the AST may be different if they do that.)

Reply:
(That situation is also fun because `Function<>` can probably be overwritten, so `getInit()` may not necessarily be the correct value of the variable.)

Reply:
I don't think so. At least we don't do that in WebKit for now.